schemas/digital-outcomes-and-services-2.declaration.json

```diff
@@ -0,0 +1,69 @@
+                "environmentalSocialLabourLaw": {"enum": [false]},
+                "bankrupt": {"enum": [false]},
+                "graveProfessionalMisconduct": {"enum": [false]},
+                "distortingCompetition": {"enum": [false]},
+                "conflictOfInterest": {"enum": [false]},
+                "distortedCompetition": {"enum": [false]},
+                "significantOrPersistentDeficiencies": {"enum": [false]},
+                "seriousMisrepresentation": {"enum": [false]},
+                "witheldSupportingDocuments": {"enum": [false]},
+                "influencedContractingAuthority": {"enum": [false]},
+                "confidentialInformation": {"enum": [false]},
+                "misleadingInformation": {"enum": [false]},
+                "unspentTaxConvictions": {"enum": [false]},
+                "GAAR": {"enum": [false]}
+            }
+        }
+    ],
+    "definitions": {
+        "notDefiniteFail": {
+            "$schema": "http://json-schema.org/draft-04/schema#",
+            "type": "object",
+            "properties": {
+                "status": {"enum": ["complete"]},
+                "termsOfParticipation": {"enum": [true]},
+                "termsAndConditions": {"enum": [true]},
+                "10WorkingDays": {"enum": [true]},
+                "informationChanges": {"enum": [true]},
+                "requisiteAuthority": {"enum": [true]},
+                "accurateInformation": {"enum": [true]},
+                "unfairCompetition": {"enum": [true]},
+                "readUnderstoodGuidance": {"enum": [true]},
+                "understandTool": {"enum": [true]},
+                "understandHowToAskQuestions": {"enum": [true]},
+                "proofOfClaims": {"enum": [true]},
+                "skillsAndResources": {"enum": [true]},
+                "accuratelyDescribed": {"enum": [true]},
+                "employersInsurance": {
+                    "type": "string",
+                    "pattern": "^(Yes |Not applicable )"
+                },
+                "offerServicesYourselves": {"enum": [true]},
+                "fullAccountability": {"enum": [true]},
+                "conspiracy": {"enum": [false]},
+                "corruptionBribery": {"enum": [false]},
+                "fraudAndTheft": {"enum": [false]},
+                "terrorism": {"enum": [false]},
+                "organisedCrime": {"enum": [false]},
+                "MI": {"enum": [true]},
+                "canProvideFromDayOne": {"enum": [true]},
+                "evidence": {"enum": [true]},
+                "environmentallyFriendly": {"enum": [true]},
+                "consistentDelivery": {"enum": [true]},
+                "skillsAndCapabilityAssessment": {"enum": [true]},
+                "continuousProfessionalDevelopment": {"enum": [true]},
+                "customerSatisfactionProcess": {"enum": [true]},
+                "equalityAndDiversity": {"enum": [true]},
+                "civilServiceValues": {"enum": [true]}
```

Comment:
Yes I appear to have missed a section. I want to get cath's input on this because the way it's been worded in the story makes it a little unclear whether these ones should be considered minimal requirements or auto-pass requirements.